packages/back-end/src/api/openapi/openapi.yaml

```diff
@@ -108,6 +108,9 @@ tags:
   - name: code-references
     x-displayName: Code References
     description: Intended for use with our code reference CI utility, [`gb-find-code-refs`](https://github.com/growthbook/gb-find-code-refs).
+  - name: ingestion
+    x-displayName: Event Tracking
+    description: For supporting tracking and enriching event data
```

Comment:
I don't think that you need this to be part of the documented public API.

Reply:
Yeah I was thinking about removing this & the tag on the new endpoint but wasn't sure what our exact stance is